docs/resources/tozny_identity.md

```diff
@@ -0,0 +1,131 @@
+# tozny_identity Resource
+
+Resource to manage an identity for a TozID Realm. Useful for supporting service users.
+
+This resource requires that the account username and password be supplied to the provider either via explicit provider settings or file based credentials.
+
+This resource currently only supports supplying a password for the user. This is saved to terraform state for use in other modules. The intent is this resource creates service
+users which are embedded in other providers and resources. At this time, it is not suggested to create identities for end users with this resource.
+
+## Example Usage
+```hcl
+# Configure the Terraform runtime
+terraform {
+  required_version = ">= 0.13"
+  required_providers {
+    tozny = {
+      # Pull signed provider binaries from
+      # the Terraform hosted registry namespace
+      # for Tozny registry.terraform.io/tozny
+      source  = "tozny/tozny"
+      # Pin Tozny provider version
+      version = ">=0.10.1"
```

Comment:
I think I generally leave the terraform block out of the docs, less strings to update when doing version updates at least ;-)